config/initializers/google_classroom_client.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "googleauth"
+require "googleauth/stores/redis_token_store"
+require "google/api_client/client_secrets"
+
+module GitHubClassroom
+  def self.google_classroom_client
+    scope = [
+      Google::Apis::ClassroomV1::AUTH_CLASSROOM_COURSES_READONLY,
+      Google::Apis::ClassroomV1::AUTH_CLASSROOM_ROSTERS_READONLY
+    ]
+
+    client_id = Rails.application.secrets.google_client_id
+    client_secret = Rails.application.secrets.google_client_secret
+    client = Google::Auth::ClientId.new(client_id, client_secret)
+
+    token_store = Google::Auth::Stores::RedisTokenStore.new(redis: GitHubClassroom.redis)
```

Comment:
Using our existing redis store to store Google Classroom user tokens

Reply:
Why did you decide to use Redis instead of Postgres?